app/controllers/publishers/vacancies/job_applications/notes_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Publishers::Vacancies::JobApplications::NotesController < Publishers::Vacancies::JobApplications::BaseController
+  def index
+    @notes_form = Publishers::JobApplication::NotesForm.new
+  end
+
+  def create
+    @notes_form = Publishers::JobApplication::NotesForm.new(notes_form_params)
+
+    if @notes_form.valid?
+      Note.create(notes_form_params.merge(job_application: job_application, publisher: current_publisher))
```

Comment:
it would be more readable if `.merge(job_application: job_application, publisher: current_publisher)` goes in `notes_form_params`

Reply:
I did try that but was causing issues. Can't remember why but I'll take another look